tools/c7n_azure/c7n_azure/resources/appserviceplan.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c7n_azure.resources.arm import ArmResourceManager
+from c7n_azure.provider import resources
+
+
+@resources.register('appserviceplan')
+class AppService(ArmResourceManager):
```

Comment:
I would make this AppServicePlan because AppService is really the new name for WebApp so its confusing.  Also the filename naming scheme would be app_service_plan.py I think.

Reply:
Thanks, meant to change that